cmd/runhcs/vm.go

```diff
@@ -52,7 +52,28 @@ var vmshimCommand = cli.Command{
 		}
 		os.Stdin.Close()
 
-		opts := &uvm.UVMOptions{}
+		var (
+			opts    interface{}
+			isLCOW  bool
+			optKeys map[string]interface{}
+		)
+
+		// Total HACK. We know that LayerFolders will only be set if we are
```

Comment:
Yuck 🤢

Reply:
Agreed going to add `--os` to the internal command. The caller already knew which ones it was.